src/coreclr/nativeaot/System.Private.Reflection.Core/src/System/Reflection/Runtime/Dispensers/DispenserThatReusesAsLongAsKeyIsAlive.cs

```diff
@@ -3,11 +3,12 @@
 
 using System;
 using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
 using System.Runtime.CompilerServices;
 
 namespace System.Reflection.Runtime.Dispensers
 {
-    internal sealed class DispenserThatReusesAsLongAsKeyIsAlive<K, V> : Dispenser<K, V>
+    internal sealed class DispenserThatReusesAsLongAsKeyIsAlive<K, [DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicParameterlessConstructor)] V> : Dispenser<K, V>
```

Comment:
@MichalStrehovsky I'm not familiar enough with nativeaot to know why this wasn't warning (in the use of `ConditionalWeakTable<K, V>` below) in nativeaot - do you have any ideas?

Reply:
IIRC warnings for generic dataflow are not implemented. I scoped out some of the warnings because it's not critical for functionality. One person working on dataflow part time on NativeAOT side is no match to several people working on this pretty much full time on the IL Linker side.

We'll be able to re-use the unit tests on the IL Linker side to bring this to parity once this becomes a priority.